app/helpers/navigation_items.rb

```diff
@@ -21,7 +21,7 @@ def for_candidate_interface(current_candidate, _current_controller)
     def for_support_primary_nav(current_support_user, current_controller)
       if current_support_user
         [
-          NavigationItem.new('Candidates', support_interface_applications_path, is_active(current_controller, %w[candidates change_course import_references application_forms ucas_matches])),
```

Comment:
Not sure we need to change the label in the primary nav: Candidates > Applications is fine.

Reply:
Hmm, I found it a bit confusing when I tried it. Mind if we keep it "Applications"? Mostly because otherwise I have to untangle the changes in this P.

Reply:
Sure. As I say, this section needs some work; the relationship between candidates and applications isn’t super clear (I’m trying to make it so, gradually)